test/plugin/scenarios/apm-toolkit-trace-scenario/config/expectedData.yaml

```diff
@@ -51,6 +51,18 @@ segmentItems:
       tags:
       - {key: testTag, value: testInfoParam}
       skipAnalysis: 'true'
+    - operationName: tool-kit-set-operation-name
+      operationId: 0,
```

Comment:
I think we don't have operation Id today.

Reply:
Yeah, I copied from ci logs. :-P

Reply:
Which CI log? This should not exist in any log, unless we miss something.

Reply:
refs:https://pipelines.actions.githubusercontent.com/xHRaUI3jotyBHERoNaT9ZLEzfRemNRrO6TJklhbAuIxWBSOXHi/_apis/pipelines/1/runs/17925/signedlogcontent/5?urlExpires=2020-04-29T09%3A33%3A27.4653961Z&urlSigningMethod=HMACV1&urlSignature=q9f6BZEI%2Be4HvY%2F7NIFuM%2FtxDU%2FpfoRP48he52KPKEU%3D

Reply:
```bash
2020-04-29T09:29:34.4401924Z     - operationName: test.org.apache.skywalking.apm.testcase.toolkit.controller.TestService.testErrorMsg()
2020-04-29T09:29:34.4402138Z       operationId: 0
2020-04-29T09:29:34.4402764Z       parentSpanId: 0
2020-04-29T09:29:34.4402912Z       spanId: 6
2020-04-29T09:29:34.4403013Z       spanLayer: Unknown
2020-04-29T09:29:34.4403820Z       startTime: 1588152568643
2020-04-29T09:29:34.4404191Z       endTime: 1588152568644
2020-04-29T09:29:34.4404321Z       componentId: 0
2020-04-29T09:29:34.4404444Z       isError: true
2020-04-29T09:29:34.4404547Z       spanType: Local
2020-04-29T09:29:34.4405405Z       peer: ''
2020-04-29T09:29:34.4405694Z       skipAnalysis: true
2020-04-29T09:29:34.4405944Z       logs:
2020-04-29T09:29:34.4406262Z       - logEvent:
2020-04-29T09:29:34.4407303Z         - {key: event, value: error}
2020-04-29T09:29:34.4407747Z         - {key: message, value: TestErrorMsg}
2020-04-29T09:29:34.4408446Z     - operationName: test.org.apache.skywalking.apm.testcase.toolkit.controller.TestService.testErrorThrowable()
2020-04-29T09:29:34.4408840Z       operationId: 0
2020-04-29T09:29:34.4409083Z       parentSpanId: 0
2020-04-29T09:29:34.4409202Z       spanId: 7
2020-04-29T09:29:34.4409317Z       spanLayer: Unknown
2020-04-29T09:29:34.4409416Z       startTime: 1588152568644
2020-04-29T09:29:34.4409792Z       endTime: 1588152568647
2020-04-29T09:29:34.4409924Z       componentId: 0
2020-04-29T09:29:34.4410005Z       isError: true
2020-04-29T09:29:34.4410206Z       spanType: Local
2020-04-29T09:29:34.4410652Z       peer: ''
2020-04-29T09:29:34.4410903Z       skipAnalysis: true
```

Reply:
Please delete all of them. @dmsolr we may need to clean more for this legacy issue.

Reply:
BTW, what the `operationId` represents for?

Reply:
I will fix it. 
The `operationId` has been removed since protocol V3.